src/secure-dfu.ts

```diff
@@ -419,27 +432,33 @@ export class SecureDfu extends EventDispatcher {
     /**
      * Sets the DFU mode of a device, preparing it for update
      * @param device The device to switch mode
+     * @param uuids Optional alternative uuids for control, packet or button
      * @returns Promise containing the device if it is still on a valid state
      */
-    public setDfuMode(device: BluetoothDevice): Promise<BluetoothDevice> {
-        return this.gattConnect(device)
+    public setDfuMode(device: BluetoothDevice, uuids?: { service?: number|string, control?: number|string, packet?: number|string, button?: number|string}): Promise<BluetoothDevice> {
```

Comment:
Please extract uuids as interface

Reply:
Do you have a preference for the name of such interface?

Reply:
hmm `Uuids` :) or perhaps `UuidOptions`

Reply:
I like UuidOptions. I'll go with that.